.github/release-drafter-config.yml

```diff
@@ -0,0 +1,48 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bug'
+autolabeler:
+  - label: 'documentation'
+    files:
+      - '*.md'
+    branch:
+      - '/docs{0,1}\/.+/'
```

Comment:
I don't think I need this option, we don't have this naming for branch

Reply:
Yeah, true. For docs, file matcher should be enough